portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro

```diff
@@ -534,9 +534,31 @@ definition {
 			locator1 = "Button#PUBLISH",
 			value1 = "Publish");
 
-		AssertElementPresent.pauseAssertVisible(locator1 = "DocumentsAndMedia#BREADCRUMB_HOME");
+		if (isSet(validationError)) {
+			var list1 = ListUtil.newListFromString("${dmDocumentFileFailed}");
+			var list2 = ListUtil.newListFromString("${validationError}");
+			var size = ListUtil.size("${list1}");
```

Comment:
@Hanlf please remember to use a space when order matters. This is happened a few times before

Reply:
@austinchiang The SF job will fail with a space here...